app/src/main/java/com/zulip/android/models/Message.java

```diff
@@ -104,7 +105,7 @@
     private String subject;
     @SerializedName("timestamp")
     @DatabaseField(columnName = TIMESTAMP_FIELD)
-    private Date timestamp;
+    private long timestamp;
```

Comment:
@vishwesh3 just trying to understand the code, can you please tell why changing the type of timestamp in message model is needed?